docs/encyclopedia/retry-policies.mdx

```diff
@@ -119,6 +119,11 @@ The wait time before a retry is the _retry interval_. A retry interval is the sm
 
 <PrettyImage src="/img/retry-interval-diagram.png" title="Diagram that shows the retry interval and its formula" />
 
+### Per-exception next retry delay
```

Comment:
```suggestion
### Per-error next retry delay
```
I think this section is language agnostic so we should use language agnostic terms